src/main/webapp/js/components/dev/ExperimentsTable/ExperimentsTable.js

```diff
@@ -0,0 +1,524 @@
+                    {
+                        var tdStatus = $(this).find(".circle");
+                        var tdStatusTitle = tdStatus.attr("title");
+                        // keep track if status is in design
+                        if (experiment.getStatus() == GEPPETTO.Resources.ExperimentStatus.COMPLETED)
+                        {
+                            tdStatus.removeClass(tdStatusTitle);
+                            tdStatus.addClass("COMPLETED");
+                            tdStatus.attr("title", "COMPLETED");
+                        } else if (experiment.getStatus() == GEPPETTO.Resources.ExperimentStatus.DELETED)
+                        {
+                            tdStatus.removeClass(tdStatusTitle);
+                            tdStatus.addClass("DELETED");
+                            tdStatus.attr("title", "DELETED");
+                        } else if (experiment.getStatus() == GEPPETTO.Resources.ExperimentStatus.RUNNING)
+                        {
+                            tdStatus.removeClass(tdStatusTitle);
+                            tdStatus.addClass("RUNNING");
+                            tdStatus.attr("title", "RUNNING");
+                        } else if (experiment.getStatus() == GEPPETTO.Resources.ExperimentStatus.DESIGN)
+                        {
+                            tdStatus.removeClass(tdStatusTitle);
+                            tdStatus.addClass("DESIGN");
+                            tdStatus.attr("title", "DESIGN");
+                        } else if (experiment.getStatus() == GEPPETTO.Resources.ExperimentStatus.QUEUED)
+                        {
+                            tdStatus.removeClass(tdStatusTitle);
+                            tdStatus.addClass("QUEUED");
+                            tdStatus.attr("title", "QUEUED");
+                        } else if (experiment.getStatus() == GEPPETTO.Resources.ExperimentStatus.CANCELED)
+                        {
+                            tdStatus.removeClass(tdStatusTitle);
+                            tdStatus.addClass("CANCELED");
+                            tdStatus.attr("title", "CANCELED");
+                        }
+                    }
+                }
+            });
+        },
+        
+		populate : function(){
+			var experiments = window.Project.getExperiments();
+			var rows = [];
+			var index = 0;
+			for ( var key in experiments)
+			{
+				var experiment = experiments[key];
+				rows[index] = experiment;
+				index++;
+			}
+			this.setState({ experiments: rows });
+		},
+
+		getInitialState: function() {
+			var tabledata = [];
+
+			return {experiments: tabledata, counter : 1};
+		},
+		onClick : function(rowID, e){
+			var targetRowId = "."+rowID;
+			if( $(targetRowId).is(':visible')){
+				$(targetRowId).hide();
+			}else{
+				$(targetRowId).show();
+			}
+		},
+		render: function() {
+			var style  = {
+					width:"30%"
```

Comment:
Curious about this extra style information -- does it not work to have this in the less file?